pkg/cable/libreswan/libreswan.go

```diff
@@ -347,8 +349,6 @@ func (i *libreswan) runPluto() error {
 	klog.Info("Starting Pluto")
 
 	args := []string{}
-	args = append(args, "--ikeport", i.ipSecIKEPort)
-	args = append(args, "--natikeport", i.ipSecNATTPort)
```

Comment:
@skitt We are removing these params from Pluto and now configuring it as part of whack.
However, for whack, we are only configuring "--ikeport". What about "--natikeport"?

Reply:
`whack` only has an `--ikeport` parameter. Pluto needed two, because the port was specified for different connection types; the daemon would then choose the appropriate port for each connection based on its type. `whack --ikeport` specifies the port for a specific connection, so there’s no longer a need to distinguish NAT and non-NAT ports.

However I suspect we’re missing something in the non-NAT case, I’ll add a comment on Miguel’s PR.

Reply:
Okay thanks for clarification @skitt